plone/session/tests/testPAS.py

```diff
@@ -133,7 +136,7 @@ def testRefresh(self):
         logout()
         session = self.folder.pas.session
         request = self.makeRequest("test string")
-        session.updateCredentials(request, request.response, "our_user", "password")
+        session._setupSession(self.userid, request.response)
```

Comment:
Wouldn't it make more sense to make this test stay logged in? I don't understand the use case of refreshing a session when there isn't an authenticated user.

Reply:
Fair point. I have updated it.  And added another test.